README.md

```diff
@@ -151,18 +151,22 @@ to match a non-virtual device with the profile.
 
 ### `interface_name`
 
-For type `ethernet`, this option restricts the profile to the
-given interface by name. This argument is optional and by default
-a profile is not restricted to any interface by name.
-Note that with [persistent interface naming](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Networking_Guide/ch-Consistent_Network_Device_Naming.html),
+For type `ethernet`, this option restricts the profile to the given interface
+by name. This argument is optional and by default the profile name is used
+unless a mac address is specified using the `mac` key. Specifying an empty
+string (`""`) allows to specify that the profile is not restricted to a network
+interface.
+
+
+!!! note With [persistent interface naming](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Networking_Guide/ch-Consistent_Network_Device_Naming.html),
```

Comment:
What do the three exclamation marks do?

Reply:
It was meant to create an admonition according to the python markdown docs but it does not seem to work in GitHub. I will change it.